examples/bri-3/src/bri/merkleTree/agents/merkleTreeStorage.agent.ts

```diff
@@ -55,6 +55,9 @@ export class MerkleTreeStorageAgent extends PrismaService {
       where: { id: merkleTree.id },
     });
 
+    // TODO: Mil5 - Mapping from a prisma client type to a domain object is not working.
```

Comment:
What was not working about this? I added in a mapping within the profile that unmarshalls the tree when its mapped. maybe something similar?

Reply:
Your approach works. We need to investigate if it is an optimal approach, since we would be introducing additional classes that represent prisma models just for the sake of mapping. Best options is if we could use prisma models directly in the mapping profiles, but i couldnt get that to work. Mil5 stuff, will be opening an issue